arm-servicebus/2014-09-01/swagger/servicebus.json

```diff
@@ -0,0 +1,937 @@
+          }
+        },
+        "sku": {
+          "$ref": "#/definitions/ARMSku",
+          "description": "Sku of the Namespace."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/NamespaceProperties",
+          "description": "Properties of the Namespace."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "description": "Description of a Namespace resource."
+    },
+    "ARMSku": {
+      "type": "object",
+      "properties": {
+        "Name": {
+          "type": "string",
+          "description": "The resource type that this object applies to. Eg: NamespaceType",
+          "enum": [
+            "Basic",
+            "Standard",
+            "Premium"
+          ],
+          "x-ms-enum": {
+            "name": "Name",
+            "modelAsString": true
+          }
+        },
+        "Tier": {
+          "type": "string",
+          "description": "The tier of this particular SKU",
+          "enum": [
+            "Basic",
+            "Standard",
+            "Premium"
+          ],
+          "x-ms-enum": {
+            "name": "Tier",
+            "modelAsString": true
+          }
+        },
+        "Capacity": {
+          "format": "int32",
+          "type": "integer",
+          "description": "The messaging units or eventhub throughput units"
+        }
+      }
+    },
+    "SharedAccessAuthorizationRuleProperties": {
+      "properties": {
+        "rights": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "enum": [
+              "Manage",
+              "Send",
+              "Listen"
+            ],
+            "x-ms-enum": {
+              "name": "AccessRights",
+              "modelAsString": false
+            }
+          },
+          "description": "The rights associated with the rule."
+        }
+      },
+      "description": "SharedAccessAuthorizationRule properties."
+    },
+    "SharedAccessAuthorizationRuleCreateOrUpdateParameters": {
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "Namespace data center location."
+        },
+        "name": {
+          "type": "string",
+          "description": "Name of the Namespace AuthorizationRule."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SharedAccessAuthorizationRuleProperties",
+          "description": "Properties of the Namespace AuthorizationRules."
+        }
+      },
+      "required": [
+        "properties"
+      ],
+      "description": "Parameters supplied to the CreateOrUpdate Namespace AuthorizationRules."
+    },
+    "SharedAccessAuthorizationRuleResource": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SharedAccessAuthorizationRuleProperties",
```

Comment:
Sibling properties of the `$ref` should actually be in the referenced definition. (Extra JSON Reference properties will be ignored)